pkgs/applications/editors/texmaker/default.nix

```diff
@@ -10,7 +10,7 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ qtbase qtscript poppler zlib ];
-  nativeBuildInputs = [ pkgconfig poppler qmake ];
+  nativeBuildInputs = [ pkgconfig poppler qmake wrapQtAppsHook ];
```

Comment:
I think you do not need both. Using Qt5's make derivation should be sufficient.

Reply:
Ok, I'll update the PR.